field_value_parser.py

```diff
@@ -14,6 +14,36 @@
 from yaml import FullLoader, load
 
 
+def _dict_key_val_str(key, value):
+	"""
+	Creates a string representing a dictionary item. The string fits this
+	format: "<key>: <value>".
+
+	Args:
+		key: the key from the dictionary item
+		value: the value from the dictionary item
+
+	Returns:
+		str: a string representing the dictionary item
+	"""
+	return str(key) + ": " + str(value)
+
+
+def _dict_key_val_type_str(key, value):
```

Comment:
Voir mon commentaire précédent sur les types

```suggestion
def _dict_key_val_type_str(key: Any, value: Any) -> str:
```